azurerm/internal/services/securitycenter/server_vulnerability_assessment_resource_test.go

```diff
@@ -0,0 +1,119 @@
+package securitycenter_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type ServerVulnerabilityAssessmentResource struct {
+}
+
+func TestAccServerVulnerabilityAssessment_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_server_vulnerability_assessment", "test")
+	r := ServerVulnerabilityAssessmentResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basicCfg(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (ServerVulnerabilityAssessmentResource) Exists(ctx context.Context, clients *clients.Client, state *terraform.InstanceState) (*bool, error) {
```

Comment:
Seems this function also should be compatible with Hybrid Machine, right?

Reply:
I have no way to automatically create a Hybrid Machine. I have only been able to verify that manually.

Reply:
Maybe you can manually create it first via [azure rest api](https://docs.microsoft.com/en-us/rest/api/hybridcompute/machines/createorupdate) and configure the resource id of this resource in test case. Then you can verify it in test case to complete the Exists function. Does it make sense?

Reply:
That will be a bit to complicated for me. But if you want to give it a go I think we can include it.